app/uk/gov/hmrc/tai/connectors/EmploymentDetailsConnector.scala

```diff
@@ -31,17 +33,20 @@ import scala.concurrent.Future
 class CachingEmploymentDetailsConnector @Inject() (
   @Named("default") underlying: EmploymentDetailsConnector,
   config: NpsConfig,
-  cachingConnector: CachingConnector
+  cachingConnector: CachingConnector,
+  crypto: ApplicationCrypto
 ) extends EmploymentDetailsConnector {
 
   override val originatorId: String = config.originatorId
   override val baseUrl: String = config.baseURL
   override def getEmploymentDetailsAsEitherT(nino: Nino, year: Int)(implicit
     hc: HeaderCarrier
-  ): EitherT[Future, UpstreamErrorResponse, HodResponse] =
+  ): EitherT[Future, UpstreamErrorResponse, HodResponse] = {
+    implicit val encrypterDecrypter: Encrypter with Decrypter = crypto.JsonCrypto
```

Comment:
this crypto points to crypto library instead of our own CryptoProvider?

Reply:
I think it's the same one. I just added a CryptoProvider but it points to the same crypto library.